portfolio/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title> ksenia iorzh junior web develorer </title>
+        <link rel="stylesheet" href="styles.css" />
+    </head>
+    <body>
+        <header>
+            <nav>
+                <ul>
+                    <li><a href="#about" id="about-link">about</a></li>
+                    <li><a href="#skills" id="skills-link">skills</a></li>
+                    <li
+                        ><a href="#portfolio" id="portfolio-link"
+                            >portfolio</a
+                        ></li
+                    >
+                    <li><a href="#contact" id="contact-link">contact</a></li>
+                </ul>
+            </nav>
+        </header>
+        <div id="title" class="title_container">
+            <h1>ksenia iorzh</h1>
+            <p>junior web developer</p>
+        </div>
+        <div id="smiley_face" class="smiley_face">
+            <img src="./images/smiley_face.svg" alt="SVG Image" />
+        </div>
+        <div id="info">
+            <div id="about" class="about_container">
+                <h2>about</h2>
+                <p
+                    >i'm a junior web developer with a curiosity for building
+                    beautiful and thoughtful experiences. as i continue to
+                    explore the ever-evolving landscape of web development, i'm
+                    eager to contribute to projects that make a meaningful
+                    impact on the digital world.
+                </p>
+            </div>
+            <div id="skills" class="skills_container">
+                <h2>skills</h2>
+                <p>HTML/CSS/JS/Figma</p>
```

Comment:
Consider adding a bit more styling to the portfolio links to make them more visually distinct.